### ISSUE TITLE
app-portage/nattka bot removing ALLARCHES keyword

### ISSUE THREAD

**Issue by ulm** [mgorny/nattka#33]
Forwarding https://bugs.gentoo.org/717634 to upstream.

Following scenario:
- Package A is added to the tree, with stabilize-allarches in metadata.
- Package B (which is stable) gains a dependency on package A and is revbumped because of its updated dependencies (B doesn't have stabilize-allarches).
- A stabilisation bug is filed for package A and its reverse dependency B.

It seems quite obvious that the bug should have an ALLARCHES keyword; if package A had been stable at the moment B was revbumped, then the new revision of B would have gone straight to stable immediately.

Apart from this, GLEP 68 says:
"zero or more <stabilize-allarches/> elements, possibly restricted to specific package versions (at most one for each version) whose presence indicates that the appropriate ebuilds are suitable for simultaneously marking stable on all architectures where a previous version is stable after arch testing on one of them (i.e. if the package is known to be fully arch-independent)."

So, the GLEP says that presence of the element indicates that ALLARCHES stabilisation is possible, but it doesn't say the absence of the element would preclude filing a bug with ALLARCHES.

tl;dr The bot should not remove the ALLARCHES keyword from a bug, if it was explicitly added by a dev.

**Comment by mgorny** [comment]
So the idea would be to add it only on transition to `sanity-check+`.
